src/bundle/Resources/public/scss/_notifications-modal.scss

```diff
@@ -112,4 +112,23 @@
             }
         }
     }
+
+    $spinner-size: 2rem;
+    &__spinner {
+        width: $spinner-size;
+        height: $spinner-size;
+        position: absolute;
+        bottom: calc(50% + #{$spinner-size/2});
+        left: calc(50% - #{$spinner-size/2});
```

Comment:
Please use `transform` CSS property with `translate()` functions instead doing such calculations. It applies both to `bottom` and `left`